corehq/apps/geospatial/templates/case_grouping_map.html

```diff
@@ -4,12 +4,26 @@
 {% load hq_shared_tags %}
 
 {% block reportcontent %}
-<div class="row panel" id="export-controls">
-  <div class="controls col-sm-2 col-md-2 col-lg-2">
-    <button class="btn-default form-control" data-bind="click: downloadCSV">
-      {% trans "Export Groups" %}
-    </button>
-  </div>
+<div class="row panel">
+  <span id="lock-groups-controls">
+    <div class="controls col-sm-2 col-md-2 col-lg-2">
+      <button data-bind="visible: showLockButton, click: toggleGroupLock" class="btn-default form-control">
```

Comment:
We can remove `showLockButton` and simply set the visible status on the `groupsLocked` observable itself. So it would be `visible: groupsLocked`.